eZ/Publish/Core/Repository/PermissionsCriterionHandler.php

```diff
@@ -92,9 +107,8 @@ public function getPermissionsCriterion($module = 'content', $function = 'read')
          * If RoleAssignment has limitation then policy OR conditions are wrapped in a AND condition with the
          * role limitation, otherwise it will be merged into RoleAssignment's OR condition.
          */
-        $currentUserRef = $this->repository->getCurrentUserReference();
+        $currentUserRef = $this->PermissionResolver->getCurrentUserReference();
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/1720/commits/c3a60b9d14944b963ca3998021c5880dfba328bc